nuget.config

```diff
@@ -1,18 +1,12 @@
 ﻿<?xml version="1.0" encoding="utf-8"?>
 <configuration>
-  <config>
-    <add key="repositoryPath" value="packages" />
-  </config>
   <packageSources>
     <!-- When <clear /> is present, previously defined sources are ignored -->
     <clear />
     <add key="nuget.org" value="https://api.nuget.org/v3/index.json" />
     <add key="Microsoft Health OSS" value="https://microsofthealthoss.pkgs.visualstudio.com/FhirServer/_packaging/Public/nuget/v3/index.json" />
   </packageSources>
-  <activePackageSource>
```

Comment:
`activePackageSource` only applies up to NuGet 2.x: https://docs.microsoft.com/en-us/nuget/reference/nuget-config-file#activepackagesource